Preliminary TCK summaries for GlassFish 6, Platform TCK and Platform …

edbratt opened the pull request:
…TCK, Web Profile.

Summaries for Platform, and Web Profile. Will refresh, once all components are finalized.

Signed-off-by: Ed Bratt <ed.bratt@oracle.com>

edbratt committed:
Preliminary TCK summaries for GlassFish 6, Platform TCK and Platform TCK, Web Profile.

Signed-off-by: Ed Bratt <ed.bratt@oracle.com>

kazumura commented on docs/website/src/main/resources/certifications/jakarta-full-profile/9.0/TCK-Results.md:
```diff
@@ -0,0 +1,595 @@
+  Jakarta Dependency Injection 2.0 TCK <br/>
+  Jakarta Contexts and Dependency Injection 3.0 TCK <br/>
+  Jakarta Bean Validation 3.0 TCK <br/>
+  Jakarta Batch 2.0 TCK tests are included in Jakarta EE 9 TCK tests, See Stage Name: jbatch <br/>
```
Do we need jbatch as the additonal requirements?  
jbatch is integerated in the platform TCK.

edbratt replied:
I believe Spec. says it is required. This clarifies how that requirement is met. We said the same thing in EE 8. I don't think this hurts anything. If you do think this causes confusion, I can remove it.

kazumura replied:
I'm confused.
I thought this section describes the list of specifications which TCKs are not included the requested specification TCK.
If all specifications need to be listed as you mentioned, EJB, Servlet, JSP, and others must be also listed.
Maybe I am wrong.

edbratt replied:
In general, all the stand-alone TCKs are actually run against Eclipse GlassFish. However, this TCK summary only lists the requirements for the Jakarta EE TCK. The Jakarta EE TCK rules list:

>     Jakarta Contexts and Dependency Injection 3.0
>     Jakarta Dependency Injection 2.0
>     Jakarta Bean Validation 3.0

I removed the reference to Batch.
Also -- I created a 9.0-RC1 sub-folder so there would be a permanent record for Specs. that will go to ballot ahead of the final EE 9 package (Interceptors, Enterprise Beans, and Managed Beans).

edbratt replied:
Also, updated links and SHA codes for Jakarta EE TCK 9.0-RC1 and GlassFish 6.0 RC2

edbratt committed:
Removed commentary about batch results based on PR comments.
Added Jakarta EE 9-RC1 for platform (necessary because the the following specifications will need a certification results to point at)
Added TCK result summaries for Enterprise Beans 4.0, Interceptors 2.0, and Managed Beans 2.0
Also, added GlassFish 6.0 RC2 to the download page.

Signed-off-by: Ed Bratt <ed.bratt@oracle.com>
